### ISSUE TITLE
version.txt

### ISSUE THREAD

**Issue by jooola** [Libresonic/libresonic#295]
I see that the [https://libresonic.org/release/version.txt](https://libresonic.org/release/version.txt) still is named SUBSONIC, should we move it to LIBRESONIC ?

**Comment by jooola** [comment]
I was looking for how can i get the version from this file to add i to the download page in the jekyll website. I didn't found nothing yet.

Should we not put this file in the website repository ?

**Comment by muff1nman** [comment]
Yes it needs to be changed, but the formatting needs to stay as is.

**Comment by EugeneKay** [comment]
This is one of the few references to SUBSONIC that I am leaving as-is unless a clear migration path can be found. A naive string-change to LIBRESONIC results in an error attempting to parse the invalid version string, which is breaking for old versions of Libresonic / Subsonic-kang.
